docs/send_tokens.md

```diff
@@ -0,0 +1,87 @@
+On this page, you can find instructions on how to send tokens using [native transfers](#native-network-transfer) and the [Inter-Blockchain Communication (IBC)](#ibc-transfer) protocol.
+
+## Native network transfer
+
+1. Ensure you are logged into the wallet.
+2. Select **Send**.
+3. Fill in the details of your transaction:
+    * **Recipient**: the address you want to send the tokens to
+    * **Token**: the token denomination or type
+    * **Amount**: the number of tokens you want to send with this transaction (you can see your current balance above the **Amount**)
+    * **Memo (Optional)**: some transactions (e.g. to/from some exchanges) require a specific memo. If not needed, you can leave it blank.
+    * **Fee**: the transaction fee. Choose from **Low**, **Average** and **High**
+    !!! tip
+        Usually, the lower the transaction fee, the longer you need to wait for your transaction to be settled on the network.
+4. Press **Send**.
+5. In the summary screen, review the details and if everything is correct, select **Approve**.
+
+!!! info
+    You can check the status of your transaction via [the explorer](https://explore-fetchhub.fetch.ai).
+
+
+## IBC transfer
+
+1. Ensure you are logged into the wallet.
+2. Make sure IBC transfers are enabled and that your selected network supports it.
+
+    !!! tip
+        To enable IBC transfers in the wallet: 
+
+        From dashboard, click the ![Hamburger icon](../images/hamburger.png) icon in the top left, then **Settings**. Toggle the **Show Advanced IBC Transfers** switch on.
+  
+           ![Enable IBC transfer](./images/enable_IBC_transfer.png)
+
+    !!! warning "Note"
+        If your selected network does not support IBC transfers, you will not see an **IBC Transfer** section in the dashboard.
+
+3. <span id="step4"></span>Click the **Transfer** button in the **IBC Transfer** section.
+
+    ![IBC Transfer section](./images/ibc_transfer.png)
+
+4. Fill in the details of your transaction:
+    * **Destination Chain**: the destination blockchain. 
+
+        !!! info
+            If you do not see your desired chain, you need to [set up IBC channels first](#first-time-origindestination-transfer).
+
+    * **Recipient**: the address you want to send the tokens to 
+    * **Memo (Optional)**: an optional memo.
+
+5. Press **Next**.
+6. In the next page, provide the following:
+    * **Token**: the denomination or type of the tokens being sent 
+    * **Amount**: the number of tokens to be sent 
+    * **Fee**: the transaction fee. Choose from **Low**, **Average**, **High**.
+7. Hit **Submit**.
+8. The wallet now shows you a summary of the transaction. Review it and if you are happy, hit **Approve** to complete the transfer.
+
+!!! warning
+    Don’t send tokens via [Interchain Transfer] directly to a CEX. In most cases, this can cause loss of your funds.
```

Comment:
@VijaySharma815

Reply:
@5A11 LGTM

Reply:
@VijaySharma815 can you clean the warning up please? 

- If `[Interchain Transfer]`is supposed to be a link then please put the link, otherwise if it is just emphasis uses bold (like rest of the pages).
- By interchain transfer do you mean IBC? if so, change it to IBC to be consistent.
- Is there a resource that explains this in more detail that we can link to?

Reply:
- [done]  If [Interchain Transfer]is supposed to be a link then please put the link, otherwise if it is just emphasis uses bold (like rest of the pages). 
- [done] By interchain transfer do you mean IBC? if so, change it to IBC to be consistent.

Reply:
Is there a resource that explains this in more detail that we can link to?

These 2 links:
- https://help.keplr.app/ibc-guides-troubleshooting/cosmos-ibc-transfers
- https://help.keplr.app/ibc-guides-troubleshooting/ibc-fa-qs

all specific for keplr wallet so I don't think it won't make more sense.